src/main/java/org/motechproject/mots/dto/InchargeDto.java

```diff
@@ -23,22 +30,30 @@
 
   @Getter
   @Setter
+  @NotEmpty(message = ValidationMessages.EMPTY_PHONE_NUMBER)
   private String phoneNumber;
 
   @Getter
   @Setter
+  @Email(message = ValidationMessages.INVALID_EMAIL)
   private String email;
 
   @Getter
   @Setter
+  @NotEmpty
+  @Uuid(message = ValidationMessages.INVALID_DISTRICT_ID)
```

Comment:
it is not required when inchange is saved, only the facilityId is

Reply:
Ok, I don't notice that. I will remove UUID and notEmpty checks